### PR TITLE
Shellcheck fixes

### PR DESCRIPTION
**Pull request by obnoxxx**
fix shellcheck inbvocation error in the Makefile and fix shellcheck errors in two scripts discovered by the test run

### CODE REVIEW

**Review by synarete — DISMISSED** [review]
Commit message have a type (finf instead of find). 
Other than that: LGTM

File: Makefile [hunk 1]
```diff
@@ -183,7 +183,7 @@ check: check-shell-scripts
 
 # rule requires shellcheck and find to run
 check-shell-scripts:
-	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find  -name '*.sh')
+	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find . -name "*.sh")
```
[the remark this answers is not in the document]
Comment (phlogistonjohn) [reply Makefile:186]: I like @synarete suggestion here, but I am also a-OK with the PR as-is. If I just approve now the PR will be automatically merged so I'll give you some time to reply to this thread to say if you want to incorporate this suggestion in this PR. If not, just say so and I'll approve and we can follow up on this idea afterwards.

[the remark this answers is not in the document]
Comment (obnoxxx) [reply Makefile]: Thanks @synarete , I added your suggestion to the PR update

**Comment by obnoxxx** [comment]
@synarete  wrote:
> Commit message have a type (finf instead of find). Other than that: LGTM

fixed the commit message typo

**Comment by phlogistonjohn** [comment]
Unfortunately, I just noticed (a new) typo in the commit message:
```
The find invocation had a syntax error, which led to the shellcheck
invocation failing due to no files provided on the command line

Signed-off-by: Michael Adam <obnox@samba.org>

sq
```
^^^ ---- this "sq" line and the blank line above it can be deleted

**Comment by obnoxxx** [comment]
> Unfortunately, I just noticed (a new) typo in the commit message:
> 
> ```
> The find invocation had a syntax error, which led to the shellcheck
> invocation failing due to no files provided on the command line
> 
> Signed-off-by: Michael Adam <obnox@samba.org>
> 
> sq
> ```
> 
> ^^^ ---- this "sq" line and the blank line above it can be deleted

fixed

**Review by phlogistonjohn — DISMISSED** [review]
LGTM, thanks

File: Makefile [hunk 2]
```diff
@@ -1,3 +1,9 @@
+SELF = $(lastword $(MAKEFILE_LIST))
+ROOT_DIR = $(realpath $(dir $(SELF)))
+
+
+
+
```
Comment (synarete) [inline Makefile:6]: Please remove those extra empty lines

**Review by synarete — DISMISSED** [review]
LGTM

File: Makefile (hunk 2 above)
Comment (obnoxxx) [reply Makefile]: done

**Review by gd — APPROVED** [review]
LGTM, RB+

**Review by anoopcs9 — APPROVED** [review]

**Comment by phlogistonjohn** [comment]
Test failure appears to be a regression in samba when built with mit krb5. I'm going to use my admin status to merge this without all tests passing.

**Comment by anoopcs9** [comment]
> Test failure appears to be a regression in samba when built with mit krb5. I'm going to use my admin status to merge this without all tests passing.

Possible workaround: https://github.com/samba-in-kubernetes/sambacc/pull/71

**Pull request merged**